app/controllers/managers_controller.rb

```diff
@@ -4,6 +4,27 @@ class ManagersController < ApplicationController
   def index
   end
 
+  def show
+    @users = User.where(manager_id: current_user.id)
+  end
+
+  def my_submitted_forms
+    @submissions = current_user.submissions.where(form_id: params[:form_id])
+  end
+
+  def show_request
+    @submissions = User.find(params[:user_id]).submissions.joins(:form).where(form: {_type: "working"})
```

Comment:
I think we have has many through here so we can get directly
user.forms.where(_type: "working")

Reply:
I think we need there the join query, will discuss